### PR TITLE
chore: release 0.33.0

### PR DESCRIPTION
**Pull request by oleobal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Update code owners (#538)
+## [0.33.0] 2022-11-22
```
Comment (ThibaultFy) [inline CHANGELOG.md:10]: Can you add the link to the futur tag ?

File: charts/substra-backend/values.yaml [hunk 2]
```diff
@@ -66,7 +66,7 @@ server:
   image:
     registry: ghcr.io
     repository: substra/substra-backend
-    tag: 0.32.0
+    tag: &backend-image-tag 0.33.0
```
Comment (ThibaultFy) [inline charts/substra-backend/values.yaml:69]: Nice :p

**Review by ThibaultFy — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (oleobal) [reply CHANGELOG.md:10]: Not sure what you mean, a link to what? The github tag URL?

Comment (ThibaultFy) [reply CHANGELOG.md:10]: Yep <https://github.com/Substra/substra-backend/releases/tag/0.33.0>

File: charts/substra-backend/values.yaml (hunk 2 above)
Comment (guilhem-barthes) [reply charts/substra-backend/values.yaml:69]: Really nice, didn't think about this use of anchor

**Review by guilhem-barthes — APPROVED** [review]

**Pull request merged**